### PR TITLE
Adding USB capability to the Vortran Laser

### PR DESCRIPTION
**Pull request by edyoshikun**
Add USB compatibility to the current vortran stardus laser

### CODE REVIEW

File: copylot/gui/_qt/custom_widgets/photom_live_window.py [hunk 1]
```diff
@@ -0,0 +1,382 @@
...
+)
+from PyQt5.QtCore import Qt, QEvent, QPoint, QRect, QPointF
+from PyQt5 import QtGui
+from copylot.gui._qt.photom_control.utils.update_dac import signal_to_dac
+
+"""
+This script creates a LiveViewWindow to control laser spot when overlaid with camera display.
+"""
+
+class LiveViewWindow(QMainWindow):
```
Comment (AhmetCanSolak) [inline copylot/gui/_qt/custom_widgets/photom_live_window.py:22]: QMainWindow is not really a QWidget as a module, feel free  to make a separate subpackage for custom_windows @edyoshikun

File: copylot/gui/_qt/photom_control/helper_functions/draw_pattern_unit.py [hunk 2]
```diff
@@ -0,0 +1,314 @@
...
+    QGridLayout,
+    QProgressBar,
+    QApplication,
+)
+
+from dac_controller.scan import DACscan
+from widgets.scan_algrthm.scan_algorithm import ScanAlgorithm
+
+
+class DrawPatternUnit(QGroupBox):
```
Comment (AhmetCanSolak) [inline copylot/gui/_qt/photom_control/helper_functions/draw_pattern_unit.py:20]: these seem  highly re-usable, feel  free to  store helper widgets in  custom_widgets

**Review by AhmetCanSolak — COMMENTED** [review]
Dropped a couple comments around file structures, didn't mean to conduct a full  review.

**Comment by edyoshikun** [comment]
this branch seems to be a copy of the photom_gui. I will re-up the vortan_usb

**Pull request closed without merging**